website/docs/Use-Cases/enhanced_inference.md

```diff
@@ -372,10 +372,84 @@ context.append(
 )
 response = client.create(context=context, messages=messages, **config)
 ```
+## Logging
 
-## Logging (for openai<1)
+When debugging or diagnosing an LLM-based system, it is often convenient to log the API calls and analyze them.
 
-When debugging or diagnosing an LLM-based system, it is often convenient to log the API calls and analyze them. `autogen.Completion` and `autogen.ChatCompletion` offer an easy way to collect the API call histories. For example, to log the chat histories, simply run:
+### For openai >= 1
```

Comment:
Would you like to add a link to the notebook in this section?

Reply:
will do